.travis.yml

```diff
@@ -20,11 +20,13 @@ jobs:
       env:
         WORDPRESS_IMAGE_VERSION: php7.2
 
+    - name: Test Release Bundle
+      script: npm run release
```

Comment:
This is a standalone script to test the packaging scripts which would previously only run during the tag build which is way too late to fail.